src/dataflow-types/src/client/controller.rs

```diff
@@ -40,12 +42,30 @@ mod storage;
 pub use compute::{ComputeController, ComputeControllerMut};
 mod compute;
 
+/// Configuration of the storage layer.
+#[derive(Debug, Clone)]
+pub enum StorageConfig {
+    /// Run a local storage instance.
+    Local,
+    /// Use a remote storage instance.
+    Remote {
+        /// The address that compute instances should connect to.
+        compute_addr: String,
+        /// The address that the controller should connect to.
+        controller_addr: String,
+        /// The number of workers in the remote process.
+        workers: usize,
```

Comment:
This configuration param .. we could have the tcp_boundary communicate the number of workers from the server to the client and not specify it on the client.

Reply:
I'll file an issue for this. Don't bother here!

Reply:
https://github.com/MaterializeInc/materialize/issues/11355

Reply:
Yes! Thanks @antiguru!